src/Symfony/Component/Console/Input/InputDefinition.php

```diff
@@ -72,8 +72,8 @@ public function setArguments(array $arguments = [])
     {
         $this->arguments = [];
         $this->requiredCount = 0;
-        $this->hasOptional = false;
-        $this->hasAnArrayArgument = false;
+        $this->lastOptional = null;
```

Comment:
Please keep the same order as the properties

Reply:
It was like that before. I just bulk changed this field using my IDE feature.

Reply:
It was like that before. I just bulk changed this field with my IDE feature.